Sources/JMESPath/Variable.swift

```diff
@@ -1,14 +1,20 @@
-import Foundation
-
-public typealias JMESArray = [Any]
-public typealias JMESObject = [String: Any]
+import Foundation  // required for JSONSerialization
```

Comment:
So we will also need to pull in [`JSONWriter`](https://github.com/swiftlang/swift-foundation/blob/main/Sources/FoundationEssentials/JSON/JSONWriter.swift)?

Reply:
Most likely